include/sentry.h

```diff
@@ -532,9 +553,9 @@ SENTRY_EXPERIMENTAL_API sentry_value_t sentry_value_new_stacktrace(
 /**
  * Sets the Stack Trace conforming to the Stack Trace Interface in a value.
  *
- * The value argument must be either an exception or thread object.
+ * The value argument must be either an exception or a thread object.
  *
- * If `ips` is NULL the current stack trace is captured, otherwise `len` stack
+ * If `ips` is NULL, the current stack trace is captured, otherwise `len` stack
```

Comment:
(nit: consistency with change above)
```suggestion
 * If `ips` is NULL, the current stack trace is captured. Otherwise, `len` stack
```